app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>Teaching computing you'll help pupils become responsible and confident users of information and technology. You'll help pupils to see and understand the world.</p>
```

Comment:
@caitlin-reidy not really sure what this 2nd sentence means? In what way/ how is this related to teaching computing?